README.md

```diff
@@ -208,6 +208,33 @@ Address::create([
 ]);
 ```
 
+#### Usage in Resource:
+To get an array form of the location casted field from the resource, you can return `parent::toArray($request)`.
```

Comment:
To get an array representation of a location-casted field from a resource, you can return...

Reply:
Done.

Reply:
perfect.